NOTLOOP_INFINITE CRINGE.py.py

```
@@ -0,0 +1,136 @@
+    tk.Button(root, text="Submit", command=check_answers, font=("Comic Sans MS", 10), bg="#FFD700").pack(pady=5)
+    result_label = tk.Label(root, text="", font=("Comic Sans MS", 10))
+    result_label.pack()
+
+    root.mainloop()
+
+def on_key_press(key):
+    try:
+        if key == keyboard.Key.backspace:
+            roast = random.choice(roasts)
+            show_notification(roast)
+    except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)